### ISSUE TITLE
Module not found: Can't resolve 'buble' - since v0.19.0

### ISSUE THREAD

**Issue by lgordey** [Rich-Harris/buble#107]
I tried to update buble version and got this error.
You can try it here https://github.com/lgordey/react-styleguidist/

I guess this related to https://github.com/Rich-Harris/buble/pull/103

**Comment by adrianheine** [comment]
What did you do? I did the following and it worked:

```
git clone https://github.com/lgordey/react-styleguidist/
cd react-styleguidist/
npm i
```

My npm is at 5.6.0.

**Comment by lgordey** [comment]
```
npm i
npm run start/build
```
I have npm `5.5.1`

**Comment by adrianheine** [comment]
Thanks for trying this out so quickly, I just released 0.19.2 which should fix this issue.
